libs/sdk-ui-theme-provider/src/cssProperties.ts

```diff
@@ -48,6 +47,15 @@ const customParserFunctions: ParserFunction[] = [
     { key: "--gd-button-borderRadius", fn: (value: string) => `${value}px` },
     { key: "--gd-button-textCapitalization", fn: (value: boolean) => (value ? "capitalize" : undefined) },
     { key: "--gd-button-dropShadow", fn: (value: boolean) => (value ? undefined : "none") },
+    {
+        key: "--gd-kpiDashboards-content-widget-borderWidth",
+        fn: (value: number) => (value ? `${value}px` : "0px"),
```

Comment:
do we want to handle default value here if we already have it in 
$gd-kpiDashboards-content-widget-borderWidth: var(
    --gd-kpiDashboards-content-widget-borderWidth,
    $item-border
);
in KD?

Reply:
If the user doesn't want to have border-width?

Reply:
then he needs to put there explicitly zero and check in function should be value !== undefined to accept 0